checker/tests/tainting/Issue1942.java

```diff
@@ -0,0 +1,18 @@
+import java.util.List;
+import org.checkerframework.checker.tainting.qual.Untainted;
+
+public class Issue1942 {
+    public interface LoadableExpression<EXPRESSION> {}
+
+    abstract static class OperatorSection<A extends LoadableExpression<A>> {
+        abstract A makeExpression(List<@Untainted A> expressions);
+    }
+
+    static class BinaryOperatorSection<B extends LoadableExpression<B>> extends OperatorSection<B> {
+        @Override
+        B makeExpression(List<@Untainted B> expressions) // <-- Error on this line
```

Comment:
I would clarify the comment and say that previously there was a false positive on that particular line.
Putting the comment on the line before would make formatting a bit nicer.